pymemcache/client/base.py

```diff
@@ -38,6 +38,13 @@
     b'cas':     (b'STORED', b'EXISTS', b'NOT_FOUND'),
 }
 
+STORE_RESULTS_BINDING = {
```

Comment:
Let's give this a slightly different name:

```suggestion
STORE_RESULTS_VALUE = {
```